data/alpha/register/registration-district.yaml

```diff
@@ -7,4 +7,4 @@ fields:
 phase: alpha
 register: registration-district
 registry: hm-passport-office
-text: Registration districts in England and Wales, defined for the purposes of the Registration Acts
+text: Registration districts in England and Wales, defined for the purposes of the Civil Registrations Acts
```

Comment:
I'm sorry, I know i'm being awfully pedantic about this lately but I think its OK the put in what you did in terms of code, but when you are changing data I think the history as to why is much more important.

i.e. `updated the description text to add clarity to which registration act we are referencing`